documentation/modules/auxiliary/dos/apple_ios/webkit_backdrop_filter_blur.md

```diff
@@ -0,0 +1,31 @@
+## Vulnerable Application
+
+This module exploits a vulnerability in Safari WebKit to crash the device.
+The bug affects all iOS devices running iOS 9 up to iOS 12 and Safari on OSX 10.13.6
+
+The device will "re-spring" the operating system, but not actually restart the device.
+
+## Verification Steps
+
+1. Start msfconsole
+1. Do: `use auxiliary/dos/apple_ios/webkit_backdrop_filter_blur`
+1. Do: `set URIPATH /` (Optional)
+1. Do: `run`
+1. When you visit the device on a vulnerable device, it should crash the operating system
```

Comment:
`When you visit the device on a vulnerable device` - sounds about right

Reply:
:trollface: